pkg/helm/helm.go

```diff
@@ -58,26 +65,61 @@ func (h *Hydrator) templateArgs(destDir string) []string {
 	if h.Version != "" {
 		args = append(args, "--version", h.Version)
 	}
-	//TODO: add logic for values/crd update.
+	if len(h.ValuesFiles) > 0 {
+		valuesFiles := strings.Split(h.ValuesFiles, ",")
+		for _, fileURL := range valuesFiles {
+			args = append(args, "-f", fileURL)
+		}
+	}
+	if len(h.ValuesKey) > 0 {
+		valuesKeys := strings.Split(h.ValuesKey, ",")
+		valuesVals := strings.Split(h.ValuesVal, ",")
+		for i, key := range valuesKeys {
+			args = append(args, "--set", key+"="+valuesVals[i])
+		}
+	}
+	includeCRDs, _ := strconv.ParseBool(h.IncludeCRDs)
+	if includeCRDs {
+		args = append(args, "--include-crds")
+	}
 	args = append(args, "--output-dir", destDir)
 	return args
 }
 
-func (h *Hydrator) registryLoginArgs() []string {
+func (h *Hydrator) registryLoginArgs(ctx context.Context) ([]string, error) {
 	args := []string{"registry", "login"}
 	if h.Auth == configsync.AuthToken {
```

Comment:
Seems more idiomatic to use `switch h.Auth` here

Reply:
Done